notebooks/pandas_penguins.py

```diff
@@ -0,0 +1,35 @@
+import marimo
```

Comment:
could we run this with `uvx marimo edit --sandbox` so the dependencies are inlined?